docs/Ferramentas de build/1-npm.md

```diff
@@ -0,0 +1,45 @@
+# NPM
+
+O npm é um gerenciador de pacotes, com ele é possível criar bibliotecas js e disponibiliza-las para a comunidade, além de poder incluir outras bibliotecas externas na sua aplicação.
+
+Não iremos nos aprofundar muito no npm nessa sessão, apenas explicar o básico de como instalar o React com a utilização do mesmo.
+
+Se quiser um estudo profundo sobre, cheque a sessão npm no [node4noobs](https://github.com/anabastos/node4noobs/blob/master/contents/1-primeiros-passos/npm.md).
```

Comment:
Ao lado do "sobre", eu colocaria "npm"
(...sobre npm...)
Sei que é implícito, mas "Explicit is better than implicit." ([Pythonistas pegaram esta piada](https://www.python.org/dev/peps/pep-0020/))

E tem sessão -> seção aqui também